sdks/python/apache_beam/utils/pipeline_options_validator_test.py

```diff
@@ -87,7 +87,7 @@ def get_validator(temp_location):
       return validator
 
     test_cases = [
-        {'temp_location': None, 'errors': ['temp_location']},
```

Comment:
No, we don't have another test suite. I expect these to happen:

staging_location: valid, temp_location: None -> no errors
staging_location: invalid, temp_location: None -> staging location invalid error
staging_location: None, temp_location: None -> Error for missing arguments staging_location and temp_location

The validator already checks whether temp_location is None or not. It is doing one thing when it is not None. If it is None, it can check whether staging_location is set or not. If not could warn that these two options are missing.